test/jdk/tools/jpackage/share/BasicTest.java

```diff
@@ -55,6 +59,29 @@
  */
 
 public final class BasicTest {
+
+    public static Collection addModulesParams() {
+        List<Object[][]> params = new ArrayList<>();
+        params.add(new Object[][] { new String[] { "ALL-DEFAULT"  } });
+        params.add(new Object[][] { new String[] { "java.desktop" } });
+        params.add(new Object[][] { new String[] { "java.desktop,jdk.jartool" } });
+        params.add(new Object[][] { new String[] { "java.desktop", "jdk.jartool" } });
+        if (isAllModulePathCapable()) {
+            params.add(new Object[][] { new String[] { "ALL-MODULE-PATH" } });
+        }
+        return Collections.unmodifiableList(params);
+    }
+
+    private static boolean isAllModulePathCapable() {
+        Path jmods = Path.of(System.getProperty("java.home"), "jmods");
+        boolean noJmods = Files.notExists(jmods);
+        if (LinkableRuntimeImage.isLinkableRuntime() && noJmods) {
+           System.out.println("ALL-MODULE-PATH test skipped for linkable run-time image");
```

Comment:
Please use `TKit.trace()` instead of `System.out.println()`

Reply:
Done.